### ISSUE TITLE
Omitting trailing slash in fiddle url leads to 404

### ISSUE THREAD

**Issue by tdreid** [facundovictor/jsfiddle-downloader#5]
I noticed that if I omit the trailing slash in a fiddle url then the file downloaded will be the jsfiddle 404 page. Example:
```
jsfiddle-downloader -l https://jsfiddle.net/tdreid/w9qg0gwh -o dlFiddle.html
Detected fiddle code = tdreid
Output file = dlFiddle.html
```
(note that 'detected fiddle code' reports the user not the fiddle)
dlFiddle.html will be...

![jsfiddle-downloader-2017-01-04-20-28-05](https://cloud.githubusercontent.com/assets/868675/21665959/ba15737c-d2bc-11e6-927d-80bffd9332fc.png)

**Comment by facundovictor** [comment]
Hi @tdreid, thanks for the report! I tested it and it downloads the fiddle. May you provide the **nodejs** version, and the **jsfiddle-downloader** version?

**Comment by tdreid** [comment]
Turns out to be Node 4.6.1 and jsfiddle-downloader 0.0.1

**Comment by facundovictor** [comment]
Fixed in version 0.1.4
